src/modules/ConstrainedCrop/Module.js

```diff
@@ -0,0 +1,56 @@
+/*
+* Crops an Image on the basic of the ratio provided
+*/
+module.exports = function ConstrainedCrop(options, UI) {
+
+  var defaults = require('./../../util/getDefaults.js')(require('./info.json'));
+  var output;
+
+  function draw(input, callback) {
+
+    var step = this,
+      startingX = Number(options.startingX || defaults.startingX),
+      startingY = Number(options.startingY || defaults.startingY),
+      constrainedRatio = (options.constrainedRatio || defaults.constrainedRatio).split(':'),
```

Comment:
Let's call it aspect ratio only. It's a well-known term. Only `AspectRatio` was not a well-known name for image processing step.